mock/example/deploy/csi-mock-driver-deployment.yaml

```diff
@@ -0,0 +1,124 @@
+# This YAML file contains the deployment of the CSIDriver
+# including node-driver-registrar, external-provisioner, external-snapshotter
+# external-resizer and mock-driver
+#
+# This YAML file also includes deployment of the CSIDriver, StorageClass
+# VolumeSnapshotClass for the mock driver.
+
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: csi-mockplugin
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: csi-mockplugin
+  template:
+    metadata:
+      labels:
+        app: csi-mockplugin
+    spec:
+      serviceAccount: csi-mock
+      containers:
+        - name: csi-provisioner
+          image: gcr.io/k8s-staging-sig-storage/csi-provisioner:v1.6.0
+          args:
+            - "--csi-address=$(ADDRESS)"
+          env:
+            - name: ADDRESS
+              value: /csi/csi.sock
+          imagePullPolicy: "IfNotPresent"
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /csi
+        - name: node-driver-registrar
+          image: gcr.io/k8s-staging-sig-storage/csi-node-driver-registrar:v1.3.0
+          args:
+            - --v=5
+            - --csi-address=/csi/csi.sock
+            - --kubelet-registration-path=/var/lib/kubelet/plugins/io.kubernetes.storage.mock/csi.sock
+          env:
+            - name: KUBE_NODE_NAME
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: spec.nodeName
+          securityContext:
+            privileged: true
+          volumeMounts:
+          - mountPath: /csi
+            name: socket-dir
+          - mountPath: /registration
+            name: registration-dir
+        - name: csi-resizer
+          image: gcr.io/k8s-staging-sig-storage/csi-resizer:v0.5.0
+          args:
+            - "--v=5"
+            - "--csi-address=$(ADDRESS)"
+          env:
+            - name: ADDRESS
+              value: /csi/csi.sock
+          securityContext:
+            privileged: true
+          volumeMounts:
+          - mountPath: /csi
+            name: socket-dir
+        - name: csi-snapshotter
+          image:  gcr.io/k8s-staging-sig-storage/csi-snapshotter:v2.1.0
+          args:
+            - "--v=5"
+            - "--csi-address=$(ADDRESS)"
+            - "--leader-election=false"
```

Comment:
I think we don't actually need to explicitly set this since the default is false